xcape/test/test_core.py

```diff
@@ -59,122 +63,67 @@ def test_calc_cape_shape_3d(p_t_td_3d, p_t_td_surface, sourcein, n_returns, use_
 decimal_zmulv = 0
 
 @pytest.mark.parametrize('use_dask', [False, True])
-def test_calc_surface_cape_model_lev(dataset_soundings, use_dask):
+@pytest.mark.parametrize('sourcein, pinc_used',
+                         [('surface', 100),('mixed-layer',1000),('most-unstable', 100)])
+@pytest.mark.parametrize('vertical_levin', ['sigma', 'pressure'])
+def test_calc_surface_cape_lev(dataset_soundings, sourcein, pinc_used, use_dask,vertical_levin):
```

Comment:
biggest change is that I created one test for all cape.

Reply:
🏆 

This is awesome!